packages/core/src/utilities/findMapKeyByValue.ts

```diff
@@ -0,0 +1,10 @@
+function findMapKeyByValue(map, searchValue) {
```

Comment:
Can you add comments on this that it will be O(n) for the number of entries?  It needs some sort of warning that it is expensive to use.